### PR TITLE
Add bold text and line breaks to Contact Form emails

### LINKED ISSUE #2642: Contact Form: improve email formatting
1. Bold font for each title
2. Space/double linebreak between each paragraph

Suggested here:
https://wordpress.org/support/topic/how-to-customize-the-sent-email?replies=3&amp;view=all#post-7357906

### PR DESCRIPTION
**Pull request by chaselivingston**
Fixes #2642

### CODE REVIEW

**Comment by chaselivingston** [comment]
Appears to be failing tests because the tests only account for plaintext without HTML formatting.

**Comment by kraftbj** [comment]
@chaselivingston I updated the tests, but it's notable that the first line is HTML but the rest seem not to be. Per the test, the Dropdown, etc lines aren't bold? I haven't tested the patch myself yet.

**Comment by chaselivingston** [comment]
@kraftbj Should be safe to update the test to include bold dropdowns, etc...

**Comment by dereksmart** [comment]
Can you please post a before/after screenshot for a designer review?

**Comment by chaselivingston** [comment]
Before: 
screen shot 2015-09-30 at 9 39 14 am

After: 
screen shot 2015-09-30 at 9 41 39 am

**Comment by justinkropp** [comment]
@chaselivingston The before looks much better. My only bit of feedback is regarding the comment area in the email. Given that the other fields have the **label** and then the text (ex: **Name:** Chase Livingston), I think it makes sense that the Comment section should follow the same pattern. In other words, instead of the comment starting on a new line, it should be:

**Comment:** Comment text goes here, etc...

Does that make sense?

**Comment by chaselivingston** [comment]
@justinkropp Here's a new after screenshot:
https://cloudup.com/cdBC5SrALFu.jpg

**Comment by justinkropp** [comment]
@chaselivingston I do like this solution more. This LGTM unless anyone else has any objections.

File: modules/contact-form/grunion-contact-form.php [hunk 1]
```diff
@@ -1471,25 +1471,31 @@ function process_submission() {
...
 		if ( !empty( $comment_author_url ) ) {
-			$message .= "$comment_author_url_label: $comment_author_url\n";
+			//$message .= "$comment_author_url_label: $comment_author_url\n";
+			$message .= sprintf ( "<b>%s:</b> %s<br /><br />", $comment_author_url_label, $comment_author_url );
 		}
 		if ( !empty( $comment_content_label ) ) {
-			$message .= "$comment_content_label: $comment_content\n";
+			//$message .= "$comment_content_label: $comment_content\n";
+			$comment_content = preg_replace( '~[\r\n]+~', '<br /><br />', $comment_content );
+            $message .= sprintf ( "<b>%s:</b> %s<br /><br />", $comment_content_label, $comment_content );
```
Comment (dereksmart) [inline modules/contact-form/grunion-contact-form.php]: indented with spaces here

File: modules/contact-form/grunion-contact-form.php [hunk 2]
```diff
@@ -1471,25 +1471,31 @@ function process_submission() {
 		$date_time_format = sprintf( $date_time_format, get_option( 'date_format' ), get_option( 'time_format' ) );
 		$time = date_i18n( $date_time_format, current_time( 'timestamp' ) );
 
-		$message = "$comment_author_label: $comment_author\n";
+		//$message = "$comment_author_label:$comment_author\n";
```
Comment (dereksmart) [inline modules/contact-form/grunion-contact-form.php]: Why are we leaving so many commented lines?

**Comment by jeherve** [comment]
@chaselivingston This will need a rebase before we can merge.

**Comment by dereksmart** [comment]
I give up on rebasing this.  It may have to be re-written.  The entire structure was changed in @zinigor's PR #2738

**Comment by zinigor** [comment]
@chaselivingston can you please test this one again? I have rebased it, and a lot of things have changed in the process.

**Comment by chaselivingston** [comment]
@zinigor The paragraph styling for comment content doesn't seem to be working any longer, new lines entered in a contact form comment aren't preserved in the email from what I can see.

**Comment by zinigor** [comment]
@chaselivingston fixed newlines, should be good to go now.

**Comment by chaselivingston** [comment]
@zinigor New lines still aren't getting preserved from the comment section, they're all getting smashed onto one line in the email.

**Comment by dereksmart** [comment]
just noting this needs to be tested on wpcom as well

**Comment by dereksmart** [comment]
tested, works well.

**Comment by sharonlaker19** [comment]
Hey! The fix seems to have reverted for some reason. User reported this in 5325894-zen.

I can replicate on my test site, and the HC that looked at the ticket before me as well. 

line breaks

Do I need to open a new bug report for this?

**Pull request merged**